samples/bookinfo/routing/mixer-rule-additional-telemetry.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: "config.istio.io/v1alpha2"
```

Comment:
The mixer rules seems to be the same and not related to v1alpha3 route rules. If so they should be treated differently:  a `miserRulesDir` variable should be defined, these rules should not be copied, and their location should not depend on the configuration (`v1alpha1` vs. `v1alpha3`). These rules should not be duplicated.

Reply:
I get your point about duplication and I don't like it either (ask @rshriram about my feelings in general on code duplication :-)). In this case I make an exception for the following reasons:
- some of the rules are actually different (use ingressgateway instead of ingress)
- the logic to select the rules that are the same will make the code less readable. We will have a situation where we mix rules from v1 and v2 dynamically in the test.
If we can live with the duplication that's what I'd stick with, but I'm open to persuasion!

Reply:
Oh, I see, the difference is in the ingress-related mixer rule, it specifies ingress gateway. You are right, since there are differences, it is better to have a v1alpha3 version of all the mixer rules.